Sources/ETCD/ETCDClient.swift

```diff
@@ -66,12 +66,13 @@ public final class EtcdClient: @unchecked Sendable {
     /// Fetch the value for a key from the ETCD server.
     ///
     /// - Parameter key: The key to fetch the value for. Parameter is of type Sequence<UInt8>.
+    /// - Parameter endKey: Optional. endKey is the key following the last key to delete for the range [key, endKey). If endKey is not given, the range is defined to contain only the key argument. If endKey is one bit larger than the given key, then the range is all the keys with the prefix (the given key). If endKey is '\0', the range is all keys greater than or equal to the key argument
     /// - Returns: A `Value` containing the fetched value, or `nil` if no value was found.
-    public func get(_ key: some Sequence<UInt8>) async throws -> Data? {
-        var rangeRequest = Etcdserverpb_RangeRequest()
-        rangeRequest.key = Data(key)
-
-        let call = client.range(rangeRequest)
+    public func get(_ key: Data, endKey: Data? = nil) async throws -> Data? {
```

Comment:
Let's just make this one `get` method that takes a `RangeRequest` parameter

Reply:
Updated!